android/java/org/chromium/chrome/browser/BraveActivity.java

```diff
@@ -99,9 +121,99 @@ public void onPause() {
         }
     }
 
+    @Override
+    public void performPostInflationStartup() {
+        super.performPostInflationStartup();
+
+        createNotificationChannel();
+        setupBraveSetDefaultBrowserNotification();
+    }
+
+    private void createNotificationChannel() {
+        Context context = ContextUtils.getApplicationContext();
+        // Create the NotificationChannel, but only on API 26+ because
+        // the NotificationChannel class is new and not in the support library
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            CharSequence name = "Brave Browser";
+            String description = "Notification channel for Brave Browser";
+            int importance = NotificationManager.IMPORTANCE_DEFAULT;
+            NotificationChannel channel = new NotificationChannel(CHANNEL_ID, name, importance);
+            channel.setDescription(description);
+            // Register the channel with the system; you can't change the importance
+            // or other notification behaviors after this
+            NotificationManager notificationManager = getSystemService(NotificationManager.class);
+            notificationManager.createNotificationChannel(channel);
+        }
+    }
+
+    private void setupBraveSetDefaultBrowserNotification() {
+        Context context = ContextUtils.getApplicationContext();
+        if (BraveSetDefaultBrowserNotificationService.isBraveSetAsDefaultBrowser(this)) {
+            // Don't ask again
+            return;
+        }
+        Intent intent = new Intent(context, BraveSetDefaultBrowserNotificationService.class);
+        context.sendBroadcast(intent);
+    }
+
     private boolean isNoRestoreState() {
         return ContextUtils.getAppSharedPreferences().getBoolean(PREF_CLOSE_TABS_ON_EXIT, false);
     }
 
+    private void handleBraveSetDefaultBrowserDialog() {
+        /* (Albert Wang): Default app settings didn't get added until API 24
+         * https://developer.android.com/reference/android/provider/Settings#ACTION_MANAGE_DEFAULT_APPS_SETTINGS
+         */
+        Intent browserIntent = new Intent("android.intent.action.VIEW", Uri.parse("http://"));
+        boolean supportsDefault = Build.VERSION.SDK_INT >= 24;
```

Comment:
Could you please change 24 to Build.VERSION_CODES.N which stands for 24, to avoid hardcoded values inside sources